engine/src/flutter/shell/platform/windows/public/flutter_windows.h

```diff
@@ -80,6 +80,13 @@ typedef struct {
 
   // GPU choice preference
   FlutterDesktopGpuPreference gpu_preference;
+
+  /// Whether to enable the Impeller rendering engine.
+  ///
+  /// When not provided, this value is treated as false. In a future release,
+  /// this setting will become a no-op when the Skia backend is fully removed.
```

Comment:
Most people don't use this C API, they use the client wrapper's C++ API.

Could we put this "In a future release" comment on the client wrapper's `DartProject::set_enable_impeller` and  `DartProject::set_enable_impeller` members too?

Reply:
I think I have to add this to both places? At any rate, added the comment to more places.